web_services/foxx/brightid/encoding.js

```diff
@@ -44,6 +44,11 @@ function hash(data) {
   return b64ToUrlSafeB64(b);
 }
 
+
+function sha256(data) {
```

Comment:
This does not make sense to have such one line function here. Caller can call `crypto.sha256` directly.

Reply:
It's done
https://github.com/BrightID/BrightID-Node/commit/b45fe046f21d108a517ccdf728b2fdd1e58adf22